python/lsst/ctrl/oods/imageData.py

```diff
@@ -38,7 +37,8 @@ def __init__(self, dataset):
         """
         self.info = {"CAMERA": "", "RAFT": "", "SENSOR": "", "OBSID": ""}
         try:
-            self.info["FILENAME"] = os.path.basename(dataset.path.ospath)
+            # self.info["FILENAME"] = os.path.basename(dataset.path.ospath)
```

Comment:
Don't leave this commented line in if it's obsolete.

Reply:
Removed